controller.js

```diff
@@ -48,6 +48,25 @@ module.exports.postComment = (request, response, next) => {
     .catch(next)
 }
 
+module.exports.patchReview = (request, response, next) => {
+    const reviewId = request.params.review_id
+    const voteAdjust = request.body.inc_votes
+
+    alterVotesByReview([reviewId, voteAdjust])
+    .then (({rows: [patchedReview]}) => {
+        response.status(200).send(patchedReview)
+    })
+    .catch(next)
+}
+
+module.exports.getUsers = (request, response, next) => {
+    fetchUsers()
+    .then(({rows}) => {
+        response.status(200).send(rows)
+    })
```

Comment:
watch out there is no catch block on this request if anything did go wrong it. would crash the server rather than send back a 500